library/stdarch/crates/core_arch/src/arm_shared/hints.rs

```diff
@@ -83,8 +83,12 @@ pub unsafe fn __sevl() {
 /// improve overall system performance.
 // Section 10.1 of ACLE says that the supported arches are: 8, 6K, 6-M
 // LLVM says "instruction requires: armv6k"
+// On ARMv6 in Thumb mode, T2 is required.
 #[cfg(any(
-    target_feature = "v6",
+    all(target_feature = "v6k", not(target_feature = "thumb-mode")),
+    target_feature = "v6t2",
+    all(target_feature = "v6", target_feature = "mclass"),
+    target_feature = "v7",
```

Comment:
Q: I believe that it is, but where would I go to verify that the `cfg` here is correct?

Reply:
Added a link to the instruction encoding page in the Arm Armv7 A.R.M.